lib/generators/settings_model/USAGE

```diff
@@ -0,0 +1,42 @@
+Description:
+    Generates a settings model.
+
+Examples:
+    * Generating a settings model (you don't need to specify the Settings::
+      namespace)
+
+        $ rails generate settings_model Notificiation
+
+        # db/migrate/<timestamp>_create_settings.rb
+        class CreateSettings < ActiveRecord::Migration[6.0]
```

Comment:
should these be called `_create_settings_notification` and `CreateSettingsNotification`

Reply:
Yes, good call, done ✅